pkg/config/config.go

```diff
@@ -894,6 +912,7 @@ type Project struct {
 	Brews           []Homebrew       `yaml:"brews,omitempty" json:"brews,omitempty"`
 	AURs            []AUR            `yaml:"aurs,omitempty" json:"aurs,omitempty"`
 	Krews           []Krew           `yaml:"krews,omitempty" json:"krews,omitempty"`
+	Ko              Ko               `yaml:"ko,omitempty" json:"ko,omitempty"`
```

Comment:
does it makes sense to allow multiple `ko`s? 

if so, we should probably start with that to prevent having to change the config later...

Reply:
I think we can do that as we accept the `workingDir` parameter to specify the exact location of the project, which includes go files. If we allow specifying multiple ko configurations, we should match them with build id's to enable consuming some build parameters as you mentioned in [here](https://github.com/goreleaser/goreleaser/pull/3490#discussion_r1006822802). WDYT?

Reply:
Yeah, I think so... 

I think we could do something like:

```
kos:
  - id: foo
    build: foo
    # other params
```

and reuse the same build configs..

Reply:
I think this design is ready with the latest changes, and we are also ready for a review; for that, I'm pinging @imjasonh. (he is going to hate me because I'm mentioning him literally everywhere 🙈)☝️